### PR TITLE
Parse multiple routes file in multi module projects Play 2.4

### LINKED ISSUE #40: Error running tests and in prod mode with version 1.5.0
I am using the 1.5.0 version as a dependency in my play project. When I run in dev mode everything is fine, but when I run in prod mode or try to run tests I get the following error:

com.google.inject.CreationException: Unable to create injector, see the following errors:

1) Error injecting constructor, java.io.FileNotFoundException: File 'conf/routes' does not exist

I have been banging my head against the wall on this for a couple of days now and cannot figure it out. 

Any hints would be appreciated.

### PR DESCRIPTION
**Pull request by Antanukas**
- Use app classloader to search for routes file. 

This might fix https://github.com/swagger-api/swagger-play/issues/40
When running in multi module project only root file is located at conf/routes. Other route files are located under separate modules and are packaged in different jars so direct path access with new File('conf/moduleA.routes') will not work. Need to use class loader.
- Parse route files of multi module projects. Parses includes of such syntax in routes file:

```
->  /serviceA   serviceA.Routes
->  /serviceB   serviceB.Routes
```

searches for files serviceA.routes and serviceB.routes using Classloader.getResource

### CODE REVIEW

**Comment by fehguy** [comment]
LGTM-- @benmccann see any issues?

File: play-2.4/swagger-play2/app/play/modules/swagger/SwaggerPlugin.scala [hunk 1]
```diff
@@ -108,13 +100,37 @@ class SwaggerPluginImpl @Inject()(lifecycle: ApplicationLifecycle, router: Route
...
+    def playRoutesClassNameToFileName(className: String) = className.replace(".Routes", ".routes")
+
+    val routesFile = config.underlying.hasPath("play.http.router") match {
+      case false => "routes"
+      case true => config.getString("play.http.router") match {
+        case None => "routes"
+        case Some(value)=> playRoutesClassNameToFileName(value)
       }
     }
+    //Parses multiple route files recursively
```
Comment (benmccann) [inline play-2.4/swagger-play2/app/play/modules/swagger/SwaggerPlugin.scala:116]: We usually put a space after a //

**Comment by benmccann** [comment]
Seems okay to me

**Comment by fehguy** [comment]
thanks!

**Pull request merged**